### PR TITLE
Fix data protection schema and documentation

### PR DESCRIPTION
**Pull request by shobha2626**
1. **What this PR does / why we need it**: Fix data protection schema and documentation

2. **Which issue(s) this PR fixes**

        (optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged):

        Fixes #

3. **Additional information**

4. **Special notes for your reviewer**

### CODE REVIEW

File: docs/resources/target_location.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ resource "tanzu-mission-control_target_location" "demo_aws_self_provisioned" {
 
 ```terraform
 resource "tanzu-mission-control_target_location" "demo_aws_self_provisioned" {
-  name          = "TARGET_LOCATION_NAME"
+  name = "TARGET_LOCATION_NAME"
 
   spec {
     target_provider = "AWS"
-    credential      = {
+    credential = {
```
Comment (ramya-bangera) [inline docs/resources/target_location.md:109]: is the formatting is reverted?

Comment (shobha2626) [reply docs/resources/target_location.md:109]: This change is as per the formatting

**Review by ankitsny — APPROVED** [review]

**Review by ankitsny — APPROVED** [review]

File: internal/resources/package/data_source_package.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ var packageSchema = map[string]*schema.Schema{
 	},
 	namespaceKey: {
 		Type:        schema.TypeString,
-		Description: "Namespae of package.",
+		Description: "Namespace of package.",
```
Comment (ramya-bangera) [inline internal/resources/package/data_source_package.go:43]: this should have the corresponding doc changes too right?

Comment (ramya-bangera) [reply internal/resources/package/data_source_package.go:43]: Ideally the schema and doc changes should go in one PR itself. Ensure to do it from next time. Approving this time considering the release timeline

**Review by ramya-bangera — APPROVED** [review]

File: internal/resources/package/data_source_package.go (hunk 2 above)
Comment (shobha2626) [reply internal/resources/package/data_source_package.go:43]: There seems to be issue with `tfplugindocs` used for the doc generation, as the description from the schema is not getting picked, instead it is removing it all.


That's the reason the doc for this is not reflecting

**Review by vmw-vjn — APPROVED** [review]

**Review by ankitsny — APPROVED** [review]

**Pull request merged**